src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -5349,62 +5293,6 @@ public static void ZeroLengthToUpper()
             Assert.Equal(expectedSource, a);
         }
 
-        [Fact]
-        public static void SameSpanToUpper()
```

Comment:
Can you keep the existing tests, and just modify them to assert that overlapping throwing, instead of removing them?

Reply:
will do

Reply:
done